ios/TwilioVoiceReactNative.m

```diff
@@ -979,6 +979,18 @@ - (void)asyncPushRegistryInitialization:(dispatch_time_t)timeout
     resolve([NSUUID UUID].UUIDString);
 }
 
+RCT_EXPORT_METHOD(voice_setIncomingCallContactHandleTemplate:(NSString *)template
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+  self.incomingCallContactHandleTemplate = template;
```

Comment:
1. should we check if the template is a valid non-empty string? probably fine with whatever the app sets
2. 4-space indentation